### ISSUE TITLE
Asus c201  "You are probably installing from scratch. Fair Enough"

### ISSUE THREAD

**Issue by zishes** [ReubenAbrams/Chrubix#9]
Hello, I've been trying to install Chrubix on my asus c201 chromebook using the instructions on http://www.chrubix.org/read-me/ for some time now and have ended up with this message: 

"cp: cannot create directory ‘/tmp/_build_here/.alarpy/.mydisk/usr/local/bin/’: No such file or directory
You are probably installing from scratch. Fair enough.
Failed to save old drivers"

My SD card is clean and I have followed the instructions perfectly, I'm wondering if there is someone with a solution out there. Would love to have the dual-boot capability and tor privacy. 

Thanks,
Zishes

**Comment by SammySezan** [comment]
Same issue here.
"cp: cannot create directory ‘/tmp/_build_here/.alarpy/.mydisk/usr/local/bin/’: No such file or directory
You are probably installing from scratch. Fair enough.
Failed to save old drivers"
I've tried 2 different 16G cards.
Samsung xe303c12-a01us
Thank you.

**Comment by ReubenAbrams** [comment]
Mea culpa... :) I walked away a year ago because no one seemed interested in the project. Perhaps it's time for me to get back into it. Is there a way to gauge the community's interest in a project such as this? I'm not looking for money or glory. I just want to see if there's a potential user base.

**Comment by ruffiano1** [comment]
There is interest right here. I'm not sure how to gauge total interest -- maybe I could post to a few subreddit's (chromebook, arm, privacy related) and see if anyone says anything.

**Comment by francois-auclair** [comment]
interested too as I use an asus flip c100 ;)

**Comment by chanelle** [comment]
Yes I have interest too! Please come back, we can make this trending in Reddit, it's relevant and seemed awesome! :-(

**Comment by DJ210** [comment]
Reuben, if you are willing to pick this back up, I will do my level best to distribute the knowledge/software with many hundreds of people that only use TOR, Pidgi, and such.

.  I ditched Windows et all about 10 years ago and worked to understand how to use Linux.  I started with a Debian distro  to access TOR , use TAILS and PGP.  I ended up with a Mint18(sarah) for everyday use.  My lappy died just as my health declined and  I am, now, only able to use a chromebook and cannot seem to get it to boot to my TAILS usb.  
I don't use public social media much.  I have been using SMF type forums for many years now and would love to share/help others to be able to use their Chromebook with privacy in the forefront.  

After Snowden and Assange et al people are starting to look towards security and encryption, albeit several years and numerous TV shows had to come first.  

I have plenty of time.  I will be glad to help in any way I can.
